### DISCUSSION TITLE
`next` release not set as latest on merge

### DISCUSSION THREAD

**Discussion by imLymei** [semantic-release/semantic-release#3449] [Q&A]
After merging `master` branch with `next`, next release is not set as latest, is it normal?

**Comment by travi** [comment]
Could you clarify, did you merge next into master with the intent of promoting to the latest/stable channel, or did you merge master into next with the intent of releasing changes from master on the next channel? Also, which merge type did you use?

**Reply by imLymei** [reply]
Hello! I merged 'next' into 'master' on two differents versions of my app, using a simple merge and a PR, intending to mark the next release as stable, but it did not happen. The next release was unmarked as a prerelease but not as the latest.
